File: src/gestorAplicacion/libreria/EjemplarRevista.java
package gestorAplicacion.libreria;

import java.io.Serializable;

public class EjemplarRevista extends Ejemplar implements Serializable {
    Revista revista;

    public EjemplarRevista(int id, EstadoEjemplar estadoEjemplar, Revista revista) {
        super(id, estadoEjemplar);
        this.revista = revista;
    }

    public Revista getRevista() {
        return revista;
    }


}

File: Python/gestorAplicacion/libreria/ejemplarRevista.py
from ejemplar import Ejemplar
from estadoEjemplar import EstadoEjemplar
from revista import Revista


class EjemplarRevista(Ejemplar):
    def __init__(self, identificador: int, estadoEjemplar: EstadoEjemplar, revista: Revista):
        super().__init__(identificador, estadoEjemplar)
        self._identificador = identificador
        self._estadoEjemplar = estadoEjemplar
        self._revista = revista

    def getRevista(self) -> Revista:
        return self._revista